core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -31,16 +35,14 @@ oppia.factory('AssetsBackendApiService', [
     var ASSET_TYPE_AUDIO = 'audio';
     var ASSET_TYPE_IMAGE = 'image';
 
+    GCS_PREFIX = ('https://storage.googleapis.com/' +
+      GLOBALS.GCS_RESOURCE_BUCKET_NAME + '/exploration/');
     var AUDIO_DOWNLOAD_URL_TEMPLATE = (
-      GLOBALS.GCS_RESOURCE_BUCKET_NAME ?
-        ('https://storage.googleapis.com/' + GLOBALS.GCS_RESOURCE_BUCKET_NAME +
-       '/exploration/<exploration_id>/assets/audio/<filename>') :
-        '/audiohandler/<exploration_id>/audio/<filename>');
+      (DEV_MODE ? '/audiohandler' : GCS_PREFIX) +
+      '/<exploration_id>/assets/audio/<filename>');
     var IMAGE_DOWNLOAD_URL_TEMPLATE = (
```

Comment:
Long run, might be good to consolidate the "dev" part into /assetsdevhandler/<exploration_id>/assets/.../<filename>. Would you agree?

If so, maybe do in a separate PR?

Reply:
I agree, so I'll create an issue and assign to myself?

Reply:
Created #5713